<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CIhmAppClientTcp</class>
 <widget class="QMainWindow" name="CIhmAppClientTcp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1085</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>appClientTcp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>353</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="laAdr">
       <property name="text">
        <string>Adresse du serveur : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="laPort">
       <property name="text">
        <string>Port du serveur :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="leAdrServeur">
       <property name="inputMask">
        <string>999.999.999.999</string>
       </property>
       <property name="text">
        <string>192.168.1.47</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lePortServeur">
       <property name="inputMask">
        <string>99999</string>
       </property>
       <property name="text">
        <string>2222</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pbConnecter">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>37</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teReception">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>1041</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leEmission">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>811</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEnvoyer">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>210</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1085</width>
     <height>44</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
